githubapp/__init__.py

```diff
@@ -37,3 +37,5 @@
     rootlogger.warn('TEST_MODE should be set to "true" or "false"')
     rootlogger.warn(e)
     TEST_MODE = False
+# Check if should add member to organization
+ADD_MEMBER = os.getenv("ADD_MEMBER", "False") == "True"
```

Comment:
```suggestion
TEST_MODE = strtobool(os.environ.get("ADD_MEMBER", "False"))
```
@kalisb would this be better? I think we'd want to allow for `0|1`, `true|false`, `TRUE|FALSE` or other casing combinations